### PR TITLE
javascript javascript2 week3

### PR DESCRIPTION
**Pull request by SvitlanaBurlaka**

### CODE REVIEW

File: javascript/javascript2/week3/Game/game.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+const restartButton = document.getElementById("restartButton");
+
+restartButton.addEventListener("click", function() {
+    pTimesPressedS.innerText = "";
+    pTimesPressedL.innerText = "";
+    countL = 0;
+    countS = 0;
+    confettiS.clear();
+    confettiL.clear();
+});
```
Comment (ElhamDargi) [inline javascript/javascript2/week3/Game/game.js:80]: Good job!!!!!!
But this code misses something. For example, when the time is finished, the counter of "S" and "L" still works. Or when you press the "Reset" button the text of "input" and "message" don't empty.
First of all, I think it's better you write the "keyDown" function in the "else" of the "Click" function and you need to stop counting before showing the result. So, you should remove the "keyDown" function.
You can do something like that:


button.addEventListener("click", function func() {
    inputValue = document.getElementById("input").value;
    if (inputValue === "") {
        message.innerText = "Oops! Remember to set a time!";
    } else {

        window.addEventListener("keydown", function countKeyDown(event) {
            setTimeout(() => {
            window.removeEventListener("keydown", countKeyDown);
            findAWinner();
        }, inputValue * 1000);

        if (event.key === "s") {
            countS += 1;
            pTimesPressedS.innerText = countS;
        }
        if (event.key === "l") {
            countL += 1;
            pTimesPressedL.innerText = countL;
        }
        });
    }
});

File: javascript/javascript2/week3/Game/game.js [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+// restarting game
+const restartButton = document.getElementById("restartButton");
+
+restartButton.addEventListener("click", function() {
+    pTimesPressedS.innerText = "";
+    pTimesPressedL.innerText = "";
+    countL = 0;
+    countS = 0;
+    confettiS.clear();
+    confettiL.clear();
```
Comment (ElhamDargi) [inline javascript/javascript2/week3/Game/game.js:77]: It's better to add these two-line for empty "message" and "Input value".

 message.innerText="";
 document.getElementById("input").value="";

File: javascript/javascript2/week3/index.js [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+let currentLocation = document.querySelector(".position ");
+
+locationButton.addEventListener("click", function() {
+    navigator.geolocation.getCurrentPosition(function(geolocationPosition) {
+        longitude = geolocationPosition.coords.longitude;
+        latitude = geolocationPosition.coords.latitude;
+        currentLocation.innerText = `This is the longitude : ${longitude}. This is the latitude: ${latitude}`;
+        initMap();
+    });
+});
```
Comment (ElhamDargi) [inline javascript/javascript2/week3/index.js:48]: Well done!!!
Very nice solution.

File: javascript/javascript2/week3/index.js [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+// Create an object that has a key whose value is a
+// function.Try calling this function.
+
+const objWithFunction = {
+    firstFunction: function squreNumbers(num) {
+        return num * num;
+    },
+};
+
+console.log(objWithFunction.firstFunction(2));
```
Comment (ElhamDargi) [inline javascript/javascript2/week3/index.js:136]: Excellent, great work!

Comment (SvitlanaBurlaka) [reply javascript/javascript2/week3/index.js:136]: Thank you so much for review!

**Pull request merged**